.github/workflows/deploy.yml

```diff
@@ -10,8 +10,8 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v1
         with:
-          node-version: 14.7
-      - run: yarn --frozen-lockfile
+          node-version: 14.x
```

Comment:
Why revert if we really require 14.7?

Reply:
From what I understand we need:
```
"^12.22.0 || ^14.17.0 || >=16.0.0". Got "14.16.1"
```

`14.7` wouldn't work.

Reply:
Right (my bad it was supposed to say 14.17). Shouldn't we use 14.17 then instead 14.x (or does x implicitly mean latest)?

Reply:
`14.x` implies the latest `v14` release. As of writing, its `14.18.1`.